organizations/organizations.go

```diff
@@ -792,3 +792,80 @@ func (oh *OrganizationHandler) UpdateOrganizationAuthentication(w http.ResponseW
 
 	utils.GetSuccess("organization settings updated successfully", nil, w)
 }
+
+// Custom Orgainization Emojis
+func (oh *OrganizationHandler) CustomEmoji(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("content-Type", "application/json")
+
+	orgID := mux.Vars(r)["id"]
+	memID := mux.Vars(r)["mem_id"]
+
+	// check that org_id is valid
+	err := ValidateOrg(orgID)
+	if err != nil {
+		utils.GetError(err, http.StatusBadRequest, w)
+		return
+	}
+
+	// check that member_id is valid
+	err = ValidateMember(orgID, memID)
+	if err != nil {
+		utils.GetError(err, http.StatusBadRequest, w)
+		return
+	}
+
+	var emoji CustomEmoji
+	err = utils.ParseJSONFromRequest(r, &emoji)
+	if err != nil {
+		utils.GetError(err, http.StatusUnprocessableEntity, w)
+		return
+	}
+
+	// fetches the details of the member uploading the emoji
+	orgMember, err := FetchMember(bson.M{"org_id": orgID, "_id": memID})
```

Comment:
Just fetch the details of the currently logged in user instead of this longer route. We can be sure that Some isn't doing the operation on behalf of another person